internals/schema_migration.py

```diff
@@ -1,175 +0,0 @@
-# Copyright 2022 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License")
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import logging
-from google.cloud import ndb  # type: ignore
-
-from framework.basehandlers import FlaskHandler
-from internals import approval_defs
-from internals.core_models import FeatureEntry, Stage
-from internals.legacy_models import Feature
-from internals.review_models import Gate, Vote
-from internals.core_enums import *
-
-
-class EvaluateGateStatus(FlaskHandler):
-
-  def get_template_data(self, **kwargs):
-    """Evaluate all existing Gate entities and set correct state."""
-    self.require_cron_header()
-
-    gates: ndb.Query = Gate.query()
-    count = 0
-    batch = []
-    BATCH_SIZE = 100
-    for gate in gates:
-      if approval_defs.update_gate_approval_state(gate):
-        batch.append(gate)
-        count += 1
-        if len(batch) > BATCH_SIZE:
-          ndb.put_multi(batch)
-          batch = []
-
-    ndb.put_multi(batch)
-    return f'{count} Gate entities updated.'
-
-
-class WriteMissingGates(FlaskHandler):
```

Comment:
We will need this when we add more gate types to backfill data on existing feature entries.